Sources/Sentry/include/SentryStacktraceBuilder.h

```diff
@@ -23,6 +23,17 @@ SENTRY_NO_INIT
  */
 - (SentryStacktrace *)buildStacktraceForCurrentThread;
 
+/**
+ * Retrieve the stacktrace for the current thread, removing frames from the SentrySDK until frames
+ * from a different package are found. When including Sentry via the Swift Package Manager the
+ * package is the same as the application that includes Sentry. In this case the full stacktrace is
+ * returned without skipping frames.
+ *
+ * @param natively Indicates whether this method should use native functions to symbolicate the
+ * frames or not. Native functions are not async safe.
+ */
+- (SentryStacktrace *)buildStacktraceForCurrentThreadNatively:(BOOL)natively;
```

Comment:
`m`: I'm a bit confused by the name of this function. Why natively? Furthermore, we always pass `YES` to this method, so we can remove the extra param.
```suggestion
- (SentryStacktrace *)buildStacktraceForCurrentThreadNatively;
```

Reply:
Because we are using OS native function to symbolicated instead of using the hand made function. Im open for suggestions on naming ;)